styles/roblox/catppuccin.user.css

```diff
@@ -0,0 +1,200 @@
+        border-color: @mantle;
+    }
+    .ropro-cloud-play-container, .container-footer,  #bodySelectorContainer, .profile-avatar-left .thumbnail-span, .btr-toggle-description{
+        background: @mantle !important;
+    }
+      #roproServerFiltersMiniButton, .server-filters-dropdown-box, #playerCountSelector, #maxPlayersListMini, #maxPlayersList, #serverRegionSelector, .server-filter-option:hover .filter-tooltip, .btr-profile .btr-games-list .btr-game-button, .dialog-container .dialog-body, .btr-game-button, .btr-game-button{
+        background: @surface0 !important;
+    }
+
+    .server-filter-option, .max-players-selection, .dialog-container .dialog-header, .notification-stream-body,.notification-stream-list, .dialog-container .dialog-input-container .dialog-input {
+        background: @surface1 !important;
+    }
+    .popover.bottom>.arrow, .popover.bottom>.arrow:after{
+        border-bottom-color: @surface0;
+    }
+
+    .btr-profile .btr-games-list .btr-game-button{
+        border-color: @surface0 !important;
+    }
+
+    .profile-avatar-left .enable-three-dee, .btr-pager-holder input.btr-pager-cur, .btr-explorer-header .btr-dropdown-container .btr-dropdown-btn .btr-dropdown-label{
+        color: @text;
+    }
+    .profile-avatar-left .enable-three-dee:hover, .profile-avatar-left .enable-three-dee:focus, .btr-bundle-animations .btr-bundle-btn, body .btr-bundle-animations .btr-bundle-btn-alt, .btr-thumb-btn-container .btr-thumb-btn, #profile-current-wearing-avatar .thumbnail-2d-container, .btr-explorer-list, .btr-properties-container, .avatar-thumbnail .toggle-three-dee, .stack .server-player-count-gauge, .profile-avatar-left .enable-three-dee{
+        border-color: @surface0;
+    }
+     .profile-avatar-right .profile-avatar-mask, .btr-pager-holder input.btr-pager-cur, .btn-generic-right-sm:focus,  .btn-generic-right-sm:hover, .btn-generic-left-sm:focus,  .btn-generic-left-sm:hover, .asset-thumb-container, .menu-vertical .menu-option:hover, item-card-container .item-card-thumb-container, .catalog-container .sticky .search-bars, .rbx-select-group .rbx-select option,  .rbx-select-group .select-option option,  .select-group .rbx-select option,  .select-group .select-option option, .dar-theme .menu-vertical .menu-option .menu-secondary-option:focus,  .menu-vertical .menu-option .menu-secondary-option:hover, .btr-catalog .btr-item-card-more, .menu-vertical .menu-option .menu-secondary-option.active,  .menu-vertical .menu-option .menu-secondary-option:active, .container-empty,  .section-content-off, .catalog-container .sticky .topic-container, .layer, .horizontal-scroller .scroller .spacer, .table, .avatar .avatar-card-image, .stack .server-player-count-gauge, .game-home-page-container .home-interleave-divider, .btr-profile-groups .game-card-thumb-container, .group-details .group-wall .input-field, .dark-theme, .light-theme, .content{
```

Comment:
I believe that this may just be an instance of old code making is way through unnoticed. I believe those elements were initially used to change the background of the webpage (not exactly sure) but whatever they did, they don't seem to do anything anymore. Will remove it.